I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Chapter_Three.rpy", line 139, in script
    jump Start_chp4 #chapter start here
ScriptError: could not find label 'Start_chp4'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Chapter_Three.rpy", line 139, in script
    jump Start_chp4 #chapter start here
  File "D:\renpy-7.3.5-sdk\renpy\ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "D:\renpy-7.3.5-sdk\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'Start_chp4'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Dracula 1.0
Tue May 19 00:39:06 2020
